cirrus/server/cirrus/feature_manifest.py

```diff
@@ -1,13 +1,42 @@
+import json
+import logging
 from typing import Any, Dict, List
 
+from fml_sdk import FmlClient, FmlError  # type: ignore
+
+from .settings import channel, fml_path
+
+logger = logging.getLogger(__name__)
+
 
 class FeatureManifestLanguage:
+    def __init__(self):
+        self.fml_client = FmlClient(fml_path, channel)
+        self.merge_errors: List[FmlError] = []
```

Comment:
this is specifically for the testing since we return the attribute .json of the merge request response from nimbus_fml, I wasn't able to write a test, I could technically write the same test you wrote, but wanted to capture the error as part of fml in the API, I am happy to explain more

Reply:
Fair enough